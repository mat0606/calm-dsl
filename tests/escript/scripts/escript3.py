# python2;success

import boto3
from google.oauth2 import service_account
tn = telnet.Telnet('www.redhat.com', 80)
print tn.read_eager()

xml_val = """<breakfast_menu>
<food>
<name>Belgian Waffles</name>
<price>$5.95</price>
<description>Two of our famous Belgian Waffles with plenty of real maple syrup</description>
<calories>650</calories>
</food>
</breakfast_menu>
"""
xml_root=xml.etree.ElementTree.fromstring(xml_val)
for i in range(1):
    print "cost of",xml_root[i][0].text,"is",xml_root[i][1].text,"with",xml_root[i][3].text,"calories"
