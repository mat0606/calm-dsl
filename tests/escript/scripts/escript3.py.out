
cost of Belgian Waffles is $5.95 with 650 calories
